spec/timestamp_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'timestamp'
+
+describe Timestamp do
+
+  it 'returns the time without seconds' do
+    expect(Timestamp.new.hour_from_now.to_s.gsub(/[1234567890]/, '*')).to eq '**:**'
```

Comment:
Interesting approach! You might get a better test if you used dependency injection here to inject `Time` into your Timestamp class. At the moment, this could fail if the method always returns `00:00` — probably not an acceptable response.